sound/soc/sof/intel/hda-dsp.c

```diff
@@ -696,12 +696,27 @@ int hda_dsp_resume(struct snd_sof_dev *sdev)
 		.state = SOF_DSP_PM_D0,
```

Comment:
Commit message:

typo: "powered down"

And please add to commit description:
""The new flow is required on Intel Tiger Lake and newer. Powering down the links is the recommended flow also for older platforms, so adjust implementation accordingly and implement only one flow.""

Reply:
@kv2019i you are quoting old code version - after the recent update `hda_trigger_ignored_streams` is no longer there.

As for the commit message, well its not that this flow is needed only for newer platforms it is needed for older ones too but here's the thing - some of them disable s0ix entry criteria in the PMC (assuming this feature is supported for that platform) and therefore can enter low power state even when i.e DMA is running.

Reply:
@mrajwa For commit msg. Yes, I know that now and thus we change the implementation for all platforms. But we should have at least a minimal explanation in commit message why this change is needed now, and why the old code was passing tests before.

Reply:
@kv2019i, good idea will update commit message to be descriptive.

Reply:
@mrajwa please also fix the several typos in the commit message. Also, it is just not descriptive of what you're altering here and why?

Reply:
Thanks for spotting typos, I have just corrected and updated commit message.